service/src/main/java/su/itpro/tasktracker/model/mapper/AssignedAccountMapper.java

```
@@ -0,0 +1,24 @@
+package su.itpro.tasktracker.model.mapper;
+
+import org.springframework.stereotype.Component;
+import su.itpro.tasktracker.model.dto.TaskAssignedDto;
+import su.itpro.tasktracker.model.entity.Account;
+import su.itpro.tasktracker.model.enums.AssignedType;
+import su.itpro.tasktracker.model.util.ProfileUtil;
+
+@Component
+public class AssignedAccountMapper implements Mapper<Account, TaskAssignedDto> {
+
+  @Override
+  public TaskAssignedDto map(Account account) {
+    if (account == null) {
+      return null;
```

лучше использовать fail fast принцип, нежели проверять и возвращать null